Cakefile

```diff
@@ -27,7 +27,8 @@ build_coffee = (callback) ->
 build_copy = () ->
     fs.writeFileSync "#{ build_dir }/README.md", fs.readFileSync("./README.md")
     fs.writeFileSync "#{ build_dir }/LICENSE", fs.readFileSync("./LICENSE")
-    fs.writeFileSync "#{ build_dir }/apiblueprint.mustache", fs.readFileSync("./apiblueprint.mustache")
+    fs.writeFileSync "#{ build_dir }/apiblueprint-request.mustache", fs.readFileSync("./apiblueprint-request.mustache")
```

Comment:
Since the template is for an API Blueprint action, this comprises the entire action including a request and response pair (transaction). Can you rename it to be `apiblueprint-action` to be consistent with API Blueprint terminology.

```suggestion
    fs.writeFileSync "#{ build_dir }/apiblueprint-action.mustache", fs.readFileSync("./apiblueprint-action.mustache")
```